### DISCUSSION TITLE
Please detach and block them

### DISCUSSION THREAD

**Discussion by mslovelymiaboo** [community/community#35176] [Other Feature Feedback, Questions, & Ideas]
Please detach from all nlockchsin mining pool they stolen my money and want return it
![F3E3D0DF-4BAA-4F78-ABC5-F2AB2E2ED1B9](https://user-images.githubusercontent.com/111466953/194013798-086d07eb-dc5a-48d7-9dbb-80e63de71802.png)
![11FD6B56-1457-4D45-B74D-6B85F7161D16](https://user-images.githubusercontent.com/111466953/194013812-a3eb023d-22af-4471-8f75-18790a87190b.png)
![DC917632-2920-497E-A570-773A3B59C8A6](https://user-images.githubusercontent.com/111466953/194013813-07ac2cdc-842c-4791-b32a-232906bbcd6b.png)
![63DC814F-7222-4669-8E5A-57CB9D6315DE](https://user-images.githubusercontent.com/111466953/194013817-b6d6a36d-019d-49e8-91a6-5ce6e866e57d.png)
![21C106EA-A760-4B80-BE01-1D075DA874EE](https://user-images.githubusercontent.com/111466953/194013818-ad6579ab-84ea-4458-a456-cec0e1229c41.png)
![C6098140-84EF-49DC-8657-0F3808D0EE31](https://user-images.githubusercontent.com/111466953/194013821-5dfdbe81-a0ba-49ab-9127-8f0dc87e1393.png)
![6330F8E8-1C03-4A8B-A95D-F8F658F3D9BE](https://user-images.githubusercontent.com/111466953/194013823-c327c015-f088-43e0-9567-b5fcbdc9476d.png)
![8013A6DD-C683-4841-9F47-76F7AA2448F6](https://user-images.githubusercontent.com/111466953/194013826-d1d2d2c4-8e13-4771-8f78-038be769fb0f.png)
![D9DB2FDE-CD0D-499D-B561-3DED7694971A](https://user-images.githubusercontent.com/111466953/194013830-029cdeb7-1277-4d1b-b491-e1a5df416bec.png)
![E91D8A6A-9D7D-4541-9E49-326719C815F6](https://user-images.githubusercontent.com/111466953/194013831-dce1c90c-aca2-40c3-b69a-9fe88cf094bd.png)
![2635E430-68C3-4889-9B08-9A3F39543DEC](https://user-images.githubusercontent.com/111466953/194013833-3fc345bf-7e47-4bb2-abdd-f9efa750af8d.png)
![04E60DDE-88FD-4F56-8004-64739E3AC28C](https://user-images.githubusercontent.com/111466953/194013835-83c8fe48-ee51-43a3-aafb-b000c33121cf.png)
![FE9CDFDA-EF0B-47E9-AA76-0E3EB401ACC5](https://user-images.githubusercontent.com/111466953/194013838-b42ca454-44c0-40b4-9ed1-b4cf0ae94ea5.png)
![E0B06593-566E-42E1-AB05-A50B2D201533](https://user-images.githubusercontent.com/111466953/194013841-4172d6a6-174c-4482-bdc1-a2272f6e49de.png)
![80459586-57B2-48E9-BEDD-0CB259E5C026](https://user-images.githubusercontent.com/111466953/194013842-dbc3c564-1e33-4c4c-8533-14340355d204.png)
![D84C98DA-C067-4A43-B924-F3B7C0B49434](https://user-images.githubusercontent.com/111466953/194013845-ba890d10-3b6c-4907-a076-6a0563633a7c.png)
![5DB89A93-C2FF-4141-A240-C21C89E6DB12](https://user-images.githubusercontent.com/111466953/194013847-3344ab64-02a7-4203-a097-95edc018b14b.png)

**Comment by queenofcorgis** [comment]
Hi @mslovelymiaboo 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
